packages/utils/node-utils/src/cleanDirectory.js

```diff
@@ -14,10 +14,10 @@
   limitations under the License.
 */
 
-import rimraf from 'rimraf';
+import fsExtra from 'fs-extra';
 
 async function cleanDirectory(dirPath) {
-  await new Promise((resolve) => rimraf(dirPath, resolve));
+  await new Promise((resolve) => fsExtra.emptyDir(dirPath, resolve));
 }
```

Comment:
It looks like fs-extra returns a promise if no callback is supplied, so we should be able to do something like:
```js
async function cleanDirectory(dirPath) {
  await fsExtra.emptyDir(dirPath);
 }
```